### PR TITLE
Buff Zombie Infection

### PR DESCRIPTION
**Pull request by liltenhead**
## About the PR
This PR changes the damage numbers of the zombie infection. It also increases the chances of getting infected per bite, with and without clothing.

## Why / Balance
Most zombies are created through people getting bit to death, or infecting already dead bodies. Wearing full clothing makes it very unlikely you'll actually get infected before you fall into a critical condition. The zombie infection doing blunt damage means you constantly bleed, making it very obvious you are infected, and makes it very easy to self treat and avoid turning as long as you have Band-Aids. Pure poison damage is much harder to self treat, and overall somewhat slows down the zombie virus from taking hold if you are receiving proper medical care.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Buffed the zombie virus to do purely poison damage, and more likely to spread the infection per bite.

### CODE REVIEW

File: Content.Server/Zombies/PendingZombieComponent.cs [hunk 1]
```diff
@@ -16,8 +16,7 @@ public sealed partial class PendingZombieComponent : Component
     {
         DamageDict = new ()
         {
-            { "Blunt", 0.25 },
-            { "Poison", 0.1 },
+            { "Poison", 0.3 },
```
Comment (Brandon-Huu) [inline Content.Server/Zombies/PendingZombieComponent.cs:19]: Why not 0.35

Comment (liltenhead) [reply Content.Server/Zombies/PendingZombieComponent.cs:19]: To slow down the infection slightly.

**Comment by IProduceWidgets** [comment]
You may want to add some brute damage back, just so the bleeding can make you actually die faster when in crit. Being crit until youre a zombie sucks.

**Comment by liltenhead** [comment]
The zombie virus does 10x damage when you're crit. You won't stay crit for long.

**Comment by IlyaElDunaev** [comment]
Good idea. Usually, players do not run away from zombies, but attack in a crowd, because infection are not dangerous.

**Comment by Killerqu00** [comment]
why not genetic damage instead of poison? feels more fitting while achieving the same purpose

**Comment by K-Dynamic** [comment]
> Good idea. Usually, players do not run away from zombies, but attack in a crowd, because infection are not dangerous.

That's because they're trying to stamp it out rather than letting it spread (and zombies tend to miss their precision attacks)

**Comment by thebadman4662** [comment]
Hmm, I wonder if perhaps sleeping should increase infection damage?

If you are sleeping on bed you will never turn from infection.
If you are sleeping on medibed you outheal the infection.

Of course being asleep means its easy to bite your neck but still...

**Pull request merged**